Library/ViewModels/PledgeViewModel.swift

```diff
@@ -117,6 +118,8 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
 
     let pledgeTotal = Signal.combineLatest(pledgeAmount, shippingCost).map(+)
 
+    self.selectedShippingAmountChanged = shippingCost
```

Comment:
So typically we'd want this sort of information to be sent via a delegate but in this case our `PledgeViewController` will have two delegates 🤔. I think this is a bit of an anti-pattern and points to a larger architectural problem that we have with our VCs talking directly to each other. I have thoughts on how we can improve this but it's outside the scope of this work.

In any case, I would suggest at least renaming this output to make what this is doing a bit clearer - something like `notifyPledgeAmountViewControllerShippingAmountChanged`.

We're likely going to want to do some refactoring here down the line but I don't want to hold this up to do that right now.

Please make this change before merging though.

Reply:
Yes, I agree with that and I think it will be a fun problem to solve. A discussion in this topic with the team would be worth it. But for now, thanks for you input 😄